frontend/ui/widget/dictquicklookup.lua

```diff
@@ -572,6 +572,7 @@ function DictQuickLookup:init()
             face = self.content_face,
             width = self.content_width,
             height = self.definition_height,
+            _dummy = true,
```

Comment:
Note sure where _dummy is really for internal use or not, but it probably isn't here :) So, drop the leading `_`?

Reply:
I meant it as a sort or "this is an internal hack and read-only, don't modify me", so I kind of like the leading `_` for that, even if it is technically set from outside sometime ^^.

Reply:
`for_measurement_only = true` feels more explicite :) `_dummy` feels a bit... dummy

Reply:
9 days later, I still think this `_dummy` could have a better/more explicite name :)

Reply:
Yup, completely slipped through the cracks ;p